src/main/java/mate/academy/bookapp/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.bookapp.controller;
+
+import java.util.List;
+import mate.academy.bookapp.dto.BookDto;
+import mate.academy.bookapp.dto.CreateBookRequestDto;
+import mate.academy.bookapp.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/books")
+public class BookController {
+    @Autowired
+    private BookService bookService;

```

Comment:
```suggestion
    private final BookService bookService;
```

and add `@RequiredArgsConstructor` under the controller class